specification/communication/Communication.Messages/client.tsp

```diff
@@ -0,0 +1,28 @@
+// client.tsp
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/versioning";
+
+import "./main.tsp";
+
+using Azure.ClientGenerator.Core;
+using TypeSpec.Versioning;
+
+@useDependency(AzureCommunicationMessagesService.Versions.c2024_02_01)
+namespace ClientForAcsMessages;
+
+@client({
+  name: "NotificationMessagesClient",
+  service: AzureCommunicationMessagesService,
+})
+interface NotificationMessagesClient extends AzureCommunicationMessagesService.NotificationMessagesOperations
+{
+  @access(Access.internal)
```

Comment:
Do we need to hide it only for C# where we replace it with a Stream type or also for other languages?

Reply:
So far we only received recommendation to use Stream in .NET and not other languages.
I've raised a question in the group chat with Azure board.